sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -294,25 +365,11 @@ static inline void gnrc_ipv6_netif_set_rtr(gnrc_ipv6_netif_t *netif, bool enable
  * @param[in] netif     The interface.
  * @param[in] enable    Status for the GNRC_IPV6_NETIF_FLAGS_RTR flag.
  */
-static inline void gnrc_ipv6_netif_set_rtr_adv(gnrc_ipv6_netif_t *netif, bool enable)
-{
-    (void)netif;    /* Don't do anything for non-routers */
-    (void)enable;
-}
-
-/**
- * @brief   Solicitates an advertisement of a neighboring router on this
- *          interface.
- *
- * @param[in] netif The interface.
- * @param[in] dst   The address of the neighboring router.
- *                  May be NULL for @ref IPV6_ADDR_ALL_ROUTERS_LINK_LOCAL.
- */
-static inline void gnrc_ipv6_netif_sol_router(gnrc_ipv6_netif_t *netif, ipv6_addr_t *dst)
-{
-    (void)netif;    /* TODO */
-    (void)dst;
-}
+void gnrc_ipv6_netif_set_rtr_adv(gnrc_ipv6_netif_t *netif, bool enable);
+#else
```

Comment:
I would've preferred the `#ifdef` in the C-File. This way it wouldn't be necessary to add an entry in the riot.doxyfile. But you know what they say.. de gustibus non est disputandum.